search-app/docs/api.md

```diff
@@ -3857,7 +3857,7 @@ curl -XDELETE -i -H "Content-Type: application/json" -H "Authorization: Bearer X
   {"concept_id": "C1200000006-PROV1"},
   {"concept_id": "C1200000007-PROV1"}]'
 
-HTTP/1.1 400 Bad Request
+HTTP/1.1 400 Bad Request 
```

Comment:
```suggestion
HTTP/1.1 200 Bad Request```

Isn't this example also `200` since the `C1200000005-PROV1` in the example would have succeeded?

Reply:
fixed